src/AaveV3_Eth_BugBounty_20230710/AaveV3_Eth_BugBounty_20230710.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {ICollector} from 'aave-address-book/common/ICollector.sol';
+
+/**
+ * @title AaveV3_Eth_BugBounty_20230710
+ * @author BGD labs
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x6f70e60abd398e1ff04ff6a78cd313b69d47df84e42b790c14c273dc5ab31674
+ * - Discussion: https://governance.aave.com/t/bgd-bug-bounties-proposal/13077
+ */
+contract AaveV3_Eth_BugBounty_20230710 {
+  ICollector public constant COLLECTOR = AaveV3Ethereum.COLLECTOR;
+  address public constant FIRST_RECIPIENT = 0xb91F64b7CD46e5cF5C0735d42D8292576aD45FAb; // Kankodu
```

Comment:
is fine, but if there is a change to name stuff more explicit would do so. Also would put things that belong contextual together closer to each other.
```
address public constant KANKUDO = ...;
uint256 public constant KANKUDO_AMOUNT = 20_000e6;
```

just because it makes it a bit easier to read & check imo.